packages/fluentui/react-northstar/src/components/SplitButton/SplitButton.tsx

```diff
@@ -112,16 +118,21 @@ class SplitButton extends AutoControlledComponent<WithAsProp<SplitButtonProps>,
     primary: customPropTypes.every([customPropTypes.disallow(['secondary']), PropTypes.bool]),
     secondary: customPropTypes.every([customPropTypes.disallow(['primary']), PropTypes.bool]),
     toggleButton: customPropTypes.itemShorthand,
+    position: PropTypes.string,
```

Comment:
Let's have a stricter typings, like in packages/fluentui/react-northstar/src/utils/positioner/types.ts lines 13 (align) and 26 (position)

Reply:
Something like:
```suggestion
    position: Position,
```
and import position from packages/fluentui/react-northstar/src/utils/positioner/types.ts

Reply:
Actually, I think more appropriate example would be packages/fluentui/react-northstar/src/components/Popup/Popup.tsx itself:

```suggestion
    position: PropTypes.oneOf(POSITIONS),
```

and import POSITIONS from '../../utils/positioner';

Reply:
Very nice suggestions, thank you @pompomon

Reply:
Done